src/settings.rs

```diff
@@ -13,7 +17,7 @@ pub struct RepositorySettings {
     pub user: String,
     pub name: String,
     pub labels: Option<Vec<String>>,
-    pub since: Option<String>,
+    pub since: Option<Since>,
```

Comment:
This is where you're defining which repository setting from `.gh-univiewer.yaml` to read in?

Reply:
Not quite sure what you mean? The structure of `RepositorySettings` does define what the hashes in the list of repository hashes in `.gh-univiewer.yaml` looks like, if that's what you mean?

Reply:
yes, that is what I mean. Words are difficult sometimes :)